docker/Makefile

```diff
@@ -5,36 +5,32 @@ all: rm-image build
 # pip. If the install process is killed midway through,
 # rerun it and check `docker stats` during execution; you may
 # need to increase Docker's memory limits.
+TAG := $(shell date +%Y-%m-%d_%H-%M-%S)
+IMAGE := "docker.io/selitvin/petastorm_ci"
 build:
 	cd .. && docker build . \
 	-f ./docker/Dockerfile \
-	--tag petastorm
-
-# Create the CI container for Travis use
-push_ci_container:
-	docker tag petastorm:latest selitvin/petastorm_ci:latest
-	echo "$$DOCKER_PASSWORD" | docker login -u "$$DOCKER_USERNAME" --password-stdin
-	docker push selitvin/petastorm_ci:latest
+	--tag $(IMAGE):$(TAG) --tag $(IMAGE):latest
+	@echo
+	@echo "You can push this image to docker.io by:"
+	@echo
+	@echo " 1. Log into docker.io:"
+	@echo
+	@echo "    docker login docker.io"
+	@echo
+	@echo " 2. Push the docker image to docker.io:"
+	@echo
+	@echo "    docker push $(IMAGE):$(TAG)"
 
 # Run a container to serve as a development workspace with
 # the local repo mounted into it. IMPORTANT: this gives
 # the container access to the host's filesystem!
 run:
 	docker run \
-	-dt \
+	-it \
```

Comment:
Do we not want to run in daemon mode anymore (`docker run -d`)?

Reply:
For debugging code in the container, I found it useful to just `make run` and not two steps, starting a daemon and then execing a shell. What do you think?